requirements.txt

```diff
@@ -30,14 +34,17 @@ psycopg2-binary==2.8.2
 py==1.8.0
 PyJWT==1.7.1
 pylint==2.3.1
+pyquery==1.4.0
```

Comment:
@bisonlou  is this library necessary for now?.

Reply:
I have no Idea how they came about. I will investigate to see if the app can run without them.